fdb_exporter.py

```diff
@@ -28,31 +28,62 @@ class FdbCollector(object):
+        yield GaugeMetricFamily('fdb_workload_operations_reads_per_second',
             'Read operations per second',
             value=data['cluster']['workload']['operations']['reads']['hz'])
 
-        yield GaugeMetricFamily('fdb_workload_operations_writes_per_second', 
+        yield GaugeMetricFamily('fdb_workload_operations_writes_per_second',
             'Total number of write operations',
             value=data['cluster']['workload']['operations']['writes']['hz'])
 
-        yield CounterMetricFamily('fdb_workload_operations_writes_total', 
+        yield CounterMetricFamily('fdb_workload_operations_writes_total',
             'Total number of write operations',
             value=data['cluster']['workload']['operations']['writes']['counter'])
 
-        yield CounterMetricFamily('fdb_workload_transactions_committed_total', 
+        yield CounterMetricFamily('fdb_workload_transactions_committed_total',
             'Total number of committed transactions',
             value=data['cluster']['workload']['transactions']['committed']['counter'])
 
-        yield CounterMetricFamily('fdb_workload_transactions_conflicted_total', 
+        yield CounterMetricFamily('fdb_workload_transactions_conflicted_total',
             'Total number of transaction conflicts',
             value=data['cluster']['workload']['transactions']['conflicted']['counter'])
 
-        yield CounterMetricFamily('fdb_workload_transactions_started_total', 
+        yield CounterMetricFamily('fdb_workload_transactions_started_total',
             'Total number of started transactions',
             value=data['cluster']['workload']['transactions']['started']['counter'])
 
+        yield GaugeMetricFamily('fdb_workload_bytes_reads_per_second',
+            'Read bytes per second',
+            value=data['cluster']['workload']['bytes']['read']['hz'])
+
+        yield GaugeMetricFamily('fdb_workload_bytes_writes_per_second',
+            'Total number of writen bytes',
+            value=data['cluster']['workload']['bytes']['written']['hz'])
+
+        yield CounterMetricFamily('fdb_workload_bytes_writes_total',
+            'Total number of writen bytes',
+            value=data['cluster']['workload']['bytes']['written']['counter'])
+
+        yield GaugeMetricFamily('fdb_latency_probe_commit_seconds',
+            'Latency commits seconds',
+            value=data['cluster']['latency_probe']['commit_seconds'])
+
+        yield GaugeMetricFamily('fdb_latency_probe_transaction_start_seconds',
+            'Latency transaction start seconds',
+            value=data['cluster']['latency_probe']['transaction_start_seconds'])
+
+        yield GaugeMetricFamily('fdb_latency_probe_read_seconds',
+            'Latency reads seconds',
+            value=data['cluster']['latency_probe']['read_seconds'])
+
+        yield GaugeMetricFamily('fdb_coordinators_quorum_state', 'Quorum status',
+            value=1 if data['client']['coordinators']['quorum_reachable'] else 0)
```

Comment:
You can convert to int rather than using a ternary statement:

    int(data['client']['coordinators']['quorum_reachable'])